go.mod

```diff
@@ -13,9 +13,11 @@ require (
 	github.com/spf13/cobra v0.0.5
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.4.0
-	golang.org/x/crypto v0.0.0-20190510104115-cbcb75029529 // indirect
```

Comment:
Not sure how we should deal with this kind of technical changes. Should we have that with some feature or fix releated changes or should we have dedicated pull requests for that ...

Reply:
that's just the cleanup of go modules which is done by `go mod tidy` I would assume ...

Reply:
result of `go mod tidy` which should be part of any development to not bring in unnecessary dependencies